about/service-objectives.md

```diff
@@ -0,0 +1,134 @@
+These are periods of time when we are less strict about adhering to the service objectives on this page.
+This ensures that our work practices are sustainable and fair for our team.
+
+Here are periods of expected reduced capacity:
+
+- Weekends
+- The first and last weeks of the year.
+- Periods of overlapping international holidays.
+
+If this is disruptive to your community's activies, please reach out and we can discuss.
+However, we encourage you to avoid planning mission-critical events or actions during periods of expected reduced capacity.
+
+:::{admonition} A note on timezones
+Remember that 2i2c's team is distributed globally, and our working time zone may be different from yours.
+We aim to have team members in time zones that are working at the same time as the communities we serve, but there may occasionally be mis-matches in working hours.
+:::
+
+(objectives:support)=
+## Support responsiveness
+
+Support is one of the most important services that 2i2c provides, especially when there are problems or outages.
+For this reason, we commit to developing a support process that is efficient in responding to issues that communities bring to us.
+We define three types of support with 2i2c:
+
+- **Incidents** are requests connected with significant degraded service for one or more communities. For example, a system-wide outage or inability of users to log-in.
+- **Change Requests** are general requests for changes or improvements to a community's hub. For example, updating the environment or improving an open source tool.
+- **Guidance Requests** are questions or requests for conversations to discuss infrastructure decisions, provide guidance, etc.
+
+Below are our objectives broken down by the type of support they relate to.
+
+:::{seealso}
+- See [](../support.md) for more information about contacting support.
+- See [](tc:support:process) for our team's support process.
+:::
+
+### General support objectives
+
+- We have a dedicated communications channel for support (see [](../support.md)).
+- At least one team member is always tasked with monitoring this channel.
+- Our support team is communicative, helpful, and [abides by our Code of Conduct](tc:code-of-conduct).
+
+### Incident support objectives
+
+Our goal is to be more rapid in responding, communicating, and resolving support requests during incidents.
+Our ability to meet these objectives will depend on the times they are reported relative to the working hours of our support team.
+
+- We will triage and respond to Incidents within 8 working hours **at most**. We will on average respond to Incidents within **2 working hours**.
+- We will prioritize resolving Incidents over any other Change requests.
+- For major or complex outages, we will re-direct capacity on our engineering team to resolve them.
+
+### Change and Guidance Request support objectives
+
+- We will triage Change and Guidance requests and respond to them within 24 working hours.
```

Comment:
If we adjust the times outlined above, these ones should be augmented accordingly, IMHO.